tests/unit/scanners/test.javascript.js

```diff
@@ -125,6 +125,39 @@ describe('JavaScript Scanner', () => {
     expect(linterMessages.length).toEqual(0);
   });
 
+  it('should scan node modules', async () => {
+    const code = oneLine`var a;`;
+
+    const jsScanner = new JavaScriptScanner(
+      code,
+      'node_modules/module/code.js'
+    );
+
+    const { linterMessages } = await jsScanner.scan();
+    expect(linterMessages.length).toEqual(0);
```

Comment:
@willdurand Thanks for double-checking this and adding new tests to cover it.

While looking to these new test cases I was wondering:
Are these test going to fail without the related fix applied to src/scanners/javascript.js?
Which kind of error they trigger in that case?

maybe we could put into code something that would trigger a validation error or warning on purpose, and then we could assert the expected linting message to be absolutely sure that the file has been scanned as expected and not been skipped. 

How that sounds to you?

Reply:
Test cases fail indeed. They raise the error mentioned in the original issue:

```
   - Array []
    + Array [
    +   Object {
    +     "code": "File ignored by default.  Use a negated ignore pattern (like \"--ignore-pattern '!<relative/path/to/filename>'\") to override.",
    +     "column": undefined,
    +     "description": null,
    +     "file": ".code.js",
    +     "line": undefined,
    +     "message": "File ignored by default.  Use a negated ignore pattern (like \"--ignore-pattern '!<relative/path/to/filename>'\") to override.",
    +     "sourceCode": undefined,
    +     "type": "warning",
    +   },
    + ]
```